src/platform/intel/cavs/alternate_reset_vector.S

```diff
@@ -0,0 +1,311 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2019 Intel Corporation. All rights reserved.
+ */
+
+#include <config.h>
+#include <sof/bit.h>
+#include <sof/common.h>
+#include <sof/drivers/interrupt.h>
+#if CONFIG_BOOT_LOADER && !CONFIG_VM_ROM
+#include <sof/lib/memory.h>
+#include <sof/lib/shim.h>
+#endif
+#include <xtensa/cacheattrasm.h>
+#include <xtensa/cacheasm.h>
+#include <xtensa/coreasm.h>
+#include <xtensa/config/system.h>
+#include <xtensa/xdm-regs.h>
+#include <xtensa/xtensa-xer.h>
+#include <xtensa/xtruntime-core-state.h>
+
+#include "xtos-internal.h"
+
+#define DSP_MS_IDC_TAR_FROM_MASTER_OFFSET 0x00
```

Comment:
A comment would be useful here too.

Reply:
All what I have is the name which is not that bad, unfortunately I didnt find any comment about this define in cavs

Reply:
Can you give me a hint, the MS_IDC_TAR_FROM_ bit make no sense.

Reply:
I have just deleted this define, in sof we have l32i without defines in most cases